src/index.html

```diff
@@ -13,10 +13,65 @@
     <title>Moyo header</title>
     <link
       rel="stylesheet"
+      crossorigin="anonymous"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      crossorigin="anonymous"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      crossorigin="anonymous"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo-link"
+      >
+        <img
+          class="logo"
+          crossorigin="anonymous"
+          src="./images/logo.png"
+          alt="logo_moyo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav_links">
+          <li class="nav_item">
```

Comment:
Remember to use semantic class names that represent the meaning of the content, not the styles or tag names. It's better to use 'nav__item' instead of 'nav_item'. (HTML Content > Class names represent the meaning of the content (not the styles or tag names))